### ISSUE TITLE
metrics not found for s3 `list buckets` operation 

### ISSUE THREAD

**Issue by sanvidhansonone** [noobaa/noobaa-core#7136]
### Environment info
- NooBaa Version: **5.12.0**
- NooBaa Image: **noobaa/noobaa-core:master-20220913**
- NooBaa Operator Image: **noobaa/noobaa-operator:5.12.0**
- Platform: **Kubernetes 1.25.4 | OpenShift 4.12.0-rc.4**

### Actual behavior
1. metrics are not reflecting in the I/O Operations for `aws s3 ls s3://` operation.
or it may be included under FS Ops metrics, but FS metrics are not getting for all operations.

### Expected behavior
1. There should be some metrics for bucket list operation.

### Steps to reproduce
1. Set up s3 account to with access key and secret key
2. perform s3 operations `aws s3 ls s3://` to get list of buckets
3. use noobaa route to check metrics on `/metrics/nsfs_stats`.

### More information - Screenshots / Logs / Other output
*List Buckets*
**aws s3 ls s3://**

![image](https://user-images.githubusercontent.com/107463477/208160235-d4ea6376-bbc9-489c-bd6a-391b7e572ebf.png)

### Please find must-gather logs:

https://ibm.ent.box.com/file/1092845779983?s=g16bxab8ujxixp9yy7q2cogh6apuiszl

**Comment by liranmauda** [comment]
@sanvidhansonone true, We did not add it, we only implemented what we discussed in the first stage.
Adding it is easy, @nimrod-becker WDYT?

**Comment by nimrod-becker** [comment]
will backport to 5.12.1
